src/ContigInterval.js

```diff
@@ -0,0 +1,41 @@
+/* @flow */
+
+var Interval = require('./Interval');
+
+/**
+ * Class representing a closed interval on the genome: contig:start-stop.
+ *
+ * The contig may be either a string ("chr22") or a number (in case the contigs
+ * are indexed, for example).
+ */
+class ContigInterval {
+  contig: string|number;
+  interval: Interval;
+
+  constructor(contig: string|number, start: number, stop: number) {
+    this.contig = contig;
+    this.interval = new Interval(start, stop);
+  }
+
+  // TODO: make these getter methods & switch to Babel.
```

Comment:
According to the [es6features repo][1], this is supposed to be valid syntax:

```javascript
class Interval {
  get length() {
    return this.stop - this.start + 1;
  }
}
```

and then you can access `length` as though it were a property, not a method:

```javascript
new Interval(0, 10).length
```

I believe this is possible in ES5 using `Object.defineProperty`, but I don't want to muck with that. `react-tools` specifically doesn't support that syntax, but Traceur & Babel do.

[1]: https://github.com/lukehoban/es6features#classes